### ISSUE TITLE
-ms- prefix not being generated

### ISSUE THREAD

**Issue by ChidoYo** [ai/autoprefixer-rails#149]
I added gem 'autoprefixer-rails' to my Gemfile and have verified that vendor prefix -webkit- is being added. But I need it to also generate the -ms- prefix because I need IE 11 to support flexbox and css-grid. 
I have found other documentation that suggests adding flex: true and grid: true but it is not very clear as to where to do this on a Rails app. Any help is appreciated.

Update: Not sure what I did but it is now generating -ms-tranform. I added a browserlist file under my stylesheets folder with the following:
> 1%
last 2 versions
IE > 8

It is still not adding the -ms- prefix to grid.

**Comment by ai** [comment]
`config/autoprefixer.yml`

Can I ask you to send PR with improved docs?

**Comment by javierjulio** [comment]
@ai I'm noticing the same issue. No -ms- prefix for grid even with the `config/autoprefixer.yml` in place with the following (which is listed in the README):
```yaml
flexbox: no-2009
browsers:
  - "> 1%"
  - "last 2 versions"
  - "IE > 8"
```
We had been using Autoprefixer-rails **without** any config but since it wasn't including the -ms- prefix for grid I wanted to add that in to see if it would.

Once I added the config, cleared cache and reloaded, I started to -ms- flex related prefixes which weren't there before but nothing for grid still. Any ideas what could be wrong?
